rest/exchange_create_withdrawal.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023-present Coinbase Global, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json, hmac, hashlib, time, requests, base64, os, sys
+from urllib.parse import urlparse
+
+API_KEY = str(os.environ.get('ACCESS_KEY'))
+PASSPHRASE = str(os.environ.get('PASSPHRASE'))
+SECRET_KEY = str(os.environ.get('SIGNING_KEY'))
+PROFILE_ID = str(os.environ.get('PROFILE_ID'))
+
+url = 'https://api.exchange.coinbase.com/withdrawals/crypto'
+
+timestamp = str(int(time.time()))
+method = 'POST'
+
+amount = '1'
+currency = 'USDC'
+crypto_address = 'crypto_address'
```

Comment:
Does this need to be an env var or CLI arg?

Reply:
Idk what that means I'm new